crt_portal/cts_forms/locale/vi/LC_MESSAGES/django.po

```diff
@@ -1284,6 +1284,10 @@ msgstr "Số hồ sơ của quý vị là:"
 msgid "Print report"
 msgstr "In baìo caìo"
 
+#: templates/forms/confirmation.html:62
+msgid "Save report"
+msgstr "Hãy biết về quyền lợi của mình"
```

Comment:
should be "save report"?

<img width="1248" alt="image" src="https://github.com/usdoj-crt/crt-portal/assets/15126660/c628a2e2-ddb7-439c-babd-d277bdb33854">

Reply:
yikes good catch! fixed!